src/main/java/com/icthh/xm/ms/configuration/service/ConfigurationService.java

```diff
@@ -154,15 +161,26 @@ public void recloneConfiguration() {
     }
 
     public ConfigurationsHashSumDto findConfigurationsHashSum(String tenant) {
-        ConfigurationList configurationList = repositoryProxy.findAll();
-        List<Configuration> actualConfigs = configurationList.getData();
+        List<Configuration> actualConfigs = getActualConfigs();
 
         return new ConfigurationsHashSumDto(actualConfigs.stream()
-            .filter(config -> config.getPath().startsWith(getTenantPathPrefix(tenant)))
+            .filter(config -> isConfigUnderTenant(config, tenant))
```

Comment:
do we have test that test isConfigUnderTenant present in this method?

Reply:
ConfigurationServiceUnitTest.findConfigurationsAll - added one more path with different tenant to test it